includes/class-wc-cache-helper.php

```diff
@@ -26,6 +20,8 @@ public static function init() {
 		add_action( 'admin_notices', array( __CLASS__, 'notices' ) );
 		add_action( 'delete_version_transients', array( __CLASS__, 'delete_version_transients' ) );
 		add_action( 'wp', array( __CLASS__, 'prevent_caching' ) );
+		add_action( 'clean_taxonomy_cache', array( __CLASS__, 'clean_taxonomy_cache' ) );
```

Comment:
Could you please explain why we need to use both `clean_taxonomy_cache` and `clean_term_cache`? When I edited a category both methods were called and so `wp_delete_cache()` was called twice for the same cache key (which is not a big deal but not necessary as well). Is there a scenario where only of one of these two methods is called?

Reply:
Just in case one is called independently of the other. Would also come in useful if we had term and taxonomy caches in the future as these methods could be reused.